pkgs/applications/misc/bazecor/default.nix

```diff
@@ -53,7 +53,7 @@ appimageTools.wrapAppImage rec {
     changelog = "https://github.com/Dygmalab/Bazecor/releases/tag/v${version}";
     sourceProvenance = [ lib.sourceTypes.binaryNativeCode ];
     license = lib.licenses.gpl3Only;
-    maintainers = with lib.maintainers; [ amesgen ];
+    maintainers = with lib.maintainers; [ amesgen gcleroux ];
```

Comment:
Note that you need to [add youself to the `maintainer-list.nix`](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md#how-to-become-a-maintainer); currently, ofborg is failing because you are not in there.

Reply:
Ah thanks, I forgot to do that part